src/globals.cpp

```diff
@@ -336,7 +336,7 @@ CTweakRef<std::string> subverOverrideTweak("net.subversionOverride",
 CTweak<bool> enableCanonicalTxOrder("consensus.enableCanonicalTxOrder",
     "True if canonical transaction ordering is enabled.  Reflects the actual state so may be switched on or off by"
     " fork time flags and blockchain reorganizations.",
-    true);
```

Comment:
(we already settle this out band I'll add here my explanation for future reference)

Have it false by default makes sense case in case someone makes an IBD with `checklevel=4` and `checkpoints=0` (i.e. full validation also for historical blocks) in fact from genesis till Nov 2018 activation ctor should not be enforced. 

Doing the other way around would mean more convoluted code IMHO.